azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -189,6 +190,9 @@ public KubernetesContainerizedImpl(final Props azkProps,
             DEFAULT_SERVICE_NAME_PREFIX);
     this.clusterName = this.azkProps.getString(ConfigurationKeys.AZKABAN_CLUSTER_NAME,
         DEFAULT_CLUSTER_NAME);
+    // This is utilized to set AZ_CLUSTER ENV variable to the POD containers.
+    this.clusterEnv = this.azkProps.getString(ConfigurationKeys.AZKABAN_CLUSTER_ENV,
```

Comment:
In what scenarios can the cluster name be different from the env?

Reply:
Environment variables are utilized within the containers, mostly by the scripts running in the container/init-container. 
Cluster Name is utilized at more than one place like pod names, pod labels, API gateway for reverse proxy, etc.

Hence the value of cluster name has restrictions like "underscore" can't be part of it. However, environment variables have no such restriction.
Moreover, the environment variable name has no dependency with pod names, pod labels, API gateway, etc. hence it can be set independently of cluster name. The default value is set to clusterName for backward compatibility.